### PR TITLE
Releasing v2.0.0-beta.6 (#1138)

### PR DESCRIPTION
**Pull request by TrevorJoelHarris**
* Releasing 2.0.0-beta.6

* Update CHANGELOG.md

Consolidating isSupported changes in CHANGELOG

* Update CHANGELOG.md

Updated formatting of Minor changes in CHANGELOG

Co-authored-by: TrevorJoelHarris <TrevorJoelHarris@users.noreply.github.com>
Co-authored-by: Trevor Harris <trharris@microsoft.com>

### CODE REVIEW

**Review by AE-MS — APPROVED** [review]
:shipit:

**Comment by erinha** [comment]
Love the consolidated isSupported change log comments. Missed one near the top about adding it to the people capability

File: packages/teams-js/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
 <!-- Start content -->
 
+## 2.0.0-beta.6
+
+Thu, 28 Apr 2022 18:25:41 GMT
+
+### Major changes
+
+- Updated `media` namespace to work as it did in v1 along with necessary changes to unit tests and teams-test-app
+- Adding isSupported check to people capability
```
Comment (erinha) [inline packages/teams-js/CHANGELOG.md:14]: Can this be added to the consolidated isSupported comments?

**Review by erinha — DISMISSED** [review]

File: packages/teams-js/CHANGELOG.md (hunk 1 above)
Comment (erinha) [reply packages/teams-js/CHANGELOG.md:14]: Agree to make this change before we release 2.0

**Review by AE-MS — APPROVED** [review]

**Review by aeunms — APPROVED** [review]
🚢

**Pull request merged**